docs/api/puppeteer.connection.session.md

```diff
@@ -8,7 +8,7 @@ sidebar_label: Connection.session
 
 ```typescript
 class Connection {
-  session(sessionId: string): CDPSession | null;
```

Comment:
Same here: We need to revert this change, as we don't expose `CdpCDPSession` publicly so there will be no way for user using Puppeteer search for it.

Reply:
Reverted